thirdparty/CMakeLists.txt

```diff
@@ -49,6 +49,7 @@ set(TP_OUTPUT ${PROJECT_SOURCE_DIR}/output)
 set_property(DIRECTORY PROPERTY EP_BASE ${TP_DIR}/build)
 
 set(OSS_URL_PREFIX "http://pegasus-thirdparties.oss-cn-beijing.aliyuncs.com")
+set(FRESH_OSS_URL_PREFIX "http://pegasus-thirdparty-package.oss-cn-beijing.aliyuncs.com")
```

Comment:
Could you please upload all thirdparties to the new bucket? And then you can replace OSS_URL_PREFIX directly.